docs/lint_errors.md

```diff
@@ -110,14 +117,61 @@ The following variables are depreciated and fail the test if they are still pres
 
 ## Error #5 - Continuous Integration configuration ## {#5}
 
-nf-core pipelines must have CI testing with Travis or Circle CI.
+nf-core pipelines must have CI testing with GitHub actions, Travis or Circle CI.
+
+This test fails if the following requirements are not met:
+
+For GitHub actions CI workflow:
+
+* `.github/workflows/ci.yml` must be turned on for `push` and `pull_request``
+* `.github/workflows/ci.yml` must test with the minimum Nextflow version specified in the pipline as `manifest.nextflowVersion` under `jobs`, `test`, `strategy`, `matrix`, `nxf_ver`. E.g.:
+
+    ```yaml
+    jobs:
+    test:
+        runs-on: ubuntu-18.04
+        strategy:
+            matrix:
+                # Nextflow versions: check pipeline minimum and current latest
+                nxf_ver: ['19.10.0', '']
+    ```
+
+* `.github/workflows/ci.yml` must pull the container with the command `docker pull <container>:dev && docker tag <container>:dev <container>:tag` under `jobs`,`test`,`steps`. E.g. for nfcore/tools container:
+
+    ```yaml
+    jobs:
+    test:
+        runs-on: ubuntu-18.04
+        steps:
+        - name: Pull image
+            run: |
+            docker pull nfcore/tools:dev && docker tag nfcore/tools:dev nfcore/tools:1.0.0
+    ```
+
+* `.github/workflows/branch.yml` must be turned on for pull requests to `master`
+* `.github/workflows/branch.yml` must check that PRs are only ok if coming from an nf-core `dev` branch or a fork `patch` branch. E.g. for nf-core/tools:
+
+    ```yaml
+    jobs:
+    test:
+    runs-on: ubuntu-18.04
+    steps:
+      # PRs are only ok if coming from an nf-core `dev` branch or a fork `patch` branch
+      - name: Check PRs
+        run: |
+          { [[ $(git remote get-url origin) == *nf-core/tools ]] && [[ ${GITHUB_HEAD_REF} = "dev" ]]; } || [[ ${GITHUB_HEAD_REF} == "patch" ]]
+    ```
+
+* `.github/workflows/linting.yml` must be turned on for push and pull requests
+* `.github/workflows/linting.yml` must perform markdown linting with the command `markdownlint ${GITHUB_WORKSPACE} -c ${GITHUB_WORKSPACE}/.github/markdownlint.yml` under `jobs`, `Markdown`, `steps`.
+* `.github/workflows/linting.yml` must perform nf-core linting with the command `nf-core lint ${GITHUB_WORKSPACE}` under `jobs`, `nf-core`, `steps`.
 
-This test fails if the following happens:
+For Travis CI:
```

Comment:
I do wonder if we should remove testing for Travis, as this will certainly drift and get out of date if we move to GitHub Actions. Basically I'd kind of prefer a clean break from Travis.

Maybe this is a fair bit of work and can wait until another release though.

Reply:
maybe this release warning people to switch to GitHub actions without failing test as we discussed with @MaxUlysse would give time for people to change to actions, and then "forcing" them the next release?